parse.py

```diff
@@ -823,7 +825,7 @@ def _to_group_name(self, field):
             elif '_' in field:
                 group = field.replace('_', '_' * n)
             else:
-                raise KeyError('duplicated group name %r' % (field, ))
+                raise KeyError('duplicated group name %r' % field)
```

Comment:
Ah, that's right.  I don't use % formatting often and forgot this weird detail.  Will change it tomorrow.